client/src/components/MeetingExperience.tsx

```diff
@@ -89,49 +88,47 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
     const locale = COMPOSITE_LOCALE_EN_US;
     const formFactorValue = new MobileDetect(window.navigator.userAgent).mobile() ? 'mobile' : 'desktop';
 
-    if (renderPostCall && postCall) {
-      return (
-        <Survey
-          postCall={postCall}
-          onRejoinCall={() => {
-            setRenderPostCall(false);
-            callWithChatAdapter.onStateChange((state: CallWithChatAdapterState) => {
-              //page is set to lobby to avoid flicker of Devices page when "rejoin call" is clicked
-              if (state.page === 'configuration') state.page = 'lobby';
-            });
-            callWithChatAdapter.joinCall();
-          }}
-        />
-      );
-    }
-
     return (
-      <CallWithChatComposite
-        adapter={callWithChatAdapter}
-        fluentTheme={fluentTheme}
-        options={{
-          callControls: {
-            chatButton: chatEnabled
-          }
-        }}
-        locale={{
-          component: locale.component,
-          strings: {
-            chat: locale.strings.chat,
-            call: {
-              ...locale.strings.call,
-              lobbyScreenWaitingToBeAdmittedTitle: waitingTitle,
-              lobbyScreenWaitingToBeAdmittedMoreDetails: waitingSubtitle
-            },
-            callWithChat: locale.strings.callWithChat
-          }
-        }}
-        icons={{
-          LobbyScreenWaitingToBeAdmitted: logo,
-          LobbyScreenConnectingToCall: logo
-        }}
-        formFactor={formFactorValue}
-      />
+      <>
+        {renderPostCall && postCall && (
+          <Survey
+            data-testid="Survey"
+            postCall={postCall}
+            onRejoinCall={async () => {
+              await callWithChatAdapter.joinCall();
+              setRenderPostCall(false);
+            }}
+          />
+        )}
+        <div style={{ display: renderPostCall ? 'none' : 'flex', flexGrow: 1, position: 'relative', height: '100%' }}>
```

Comment:
We should match the conditions here as line 93 since the visibility of Survey and CallWithChatComposite are mutually exclusive. `renderPostCall && postCall`.

Reply:
renderPostCall is false by default, gets set to true only if postCall.survey.type has a value. The reason the "&&postCall" check is added was because postCall is an optinal prop. this is not needed at this point and is not being used for the render of CallWithChatComposite